tests/Repository.py

```diff
@@ -2275,3 +2275,24 @@ def testGetVulnerabilityAlertWhenTurnedOff(self):
     def testRequester(self):
         lazy_repo = self.getLazyRepository()
         assert lazy_repo.requester is lazy_repo._requester
+
+    def testGetReleases(self):
+        lazy_repo = self.getLazyRepository()
+        releases = lazy_repo.get_releases()
+        self.assertEqual(releases.totalCount, 2)
+        self.assertListEqual([release.tag_name for release in releases], ["v1.55", "v1.50"])
```

Comment:
If we swap these lines, we should be able to get away with one request (
```suggestion
        self.assertListEqual([release.tag_name for release in releases], ["v1.55", "v1.50"])
        self.assertEqual(releases.totalCount, 2)
```